index.d.ts

```diff
@@ -464,8 +464,2262 @@ interface Typography {
+    gray600: string
+    gray500: string
+    gray400: string
+    gray300: string
+    gray200: string
+    gray100: string
+    gray90: string
+    gray75: string
+    gray50: string
+    blue900: string
+    blue800: string
+    blue700: string
+    blue600: string
+    blue500: string
+    blue400: string
+    blue300: string
+    blue200: string
+    blue100: string
+    blue50: string
+    blue25: string
+    red700: string
+    red600: string
+    red500: string
+    red300: string
+    red100: string
+    red25: string
+    green900: string
+    green800: string
+    green700: string
+    green600: string
+    green500: string
+    green400: string
+    green300: string
+    green200: string
+    green100: string
+    green25: string
+    orange700: string
+    orange500: string
+    orange100: string
+    orange25: string
+    purple600: string
+    purple100: string
+    teal800: string
+    teal100: string
+    yellow800: string
+    yellow100: string
+    muted: string
+    default: string
+    dark: string
+    selected: string
+    tint1: string
+    tint2: string
+    overlay: string
+    yellowTint: string
+    greenTint: string
+    orangeTint: string
+    redTint: string
+    blueTint: string
+    purpleTint: string
+    tealTint: string
+    border: { default: string; muted: string }
+    icon: {
+      default: string
+      muted: string
+      disabled: string
+      selected: string
+    }
+    text: { danger: string; success: string; info: string }
+  }
+  fills: {
+    neutral: { color: string; backgroundColor: string }
+    blue: { color: string; backgroundColor: string }
+    red: { color: string; backgroundColor: string }
+    orange: { color: string; backgroundColor: string }
+    yellow: { color: string; backgroundColor: string }
+    green: { color: string; backgroundColor: string }
+    teal: { color: string; backgroundColor: string }
+    purple: { color: string; backgroundColor: string }
+  }
+  intents: {
+    info: {
+      background: string
+      border: string
+      text: string
+      icon: string
+    }
+    success: {
+      background: string
+      border: string
+      text: string
+      icon: string
+    }
+    warning: {
+      background: string
+      border: string
+      text: string
+      icon: string
+    }
+    danger: {
+      background: string
+      border: string
+      text: string
+      icon: string
+    }
+  }
+  radii: string[]
+  shadows: string[]
+  fontFamilies: {
+    display: string
+    mono: string
+    ui: string
+  }
+  fontSizes: string[]
+  fontWeights: { light: number; normal: number; semibold: number; bold: number }
+  letterSpacings: {
+    tightest: string
+    tighter: string
+    tight: string
+    normal: string
+    wide: string
+  }
+  lineHeights: string[]
+  zIndices: {
+    focused: number
+    stack: number
+    positioner: number
+    overlay: number
+    toaster: number
+  }
+  components: {
+    Alert: { appearances: {}; sizes: {} }
```

Comment:
In my personal project, I opted to leave out the `components` types since I didn't really need them and they added a lot of noise. Since these are more likely to change & will in turn be harder to keep in sync with the actual component styles until we're either ported to TypeScript or have a codegen solution, I am leaning on the side of removing these, but I would love to hear others' thoughts